src/redux/greeting/greettingSlice.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+// eslint-disable-next-line consistent-return
+export const fetchGreeting = createAsyncThunk('greeting/fetchGreeting', async () => {
+  try {
+    const response = await fetch('/greetings');
+    const data = await response.json();
+    return data.greeting;
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error);
+  }
```

Comment:
- [ ] Great strides in your implementation! 👏 However, I'd recommend against disabling linter rules. These rules are crafted to guide you in writing clean, industry-standard code. Instead of bypassing them, it's beneficial to address and fix any reported errors.

- [ ] Additionally, let's enhance code cleanliness by removing all `console logs`. Relying on console logs, especially in production code, isn't a recommended practice. Instead, consider managing errors through a dedicated state, allowing for more controlled handling within your `extraReducers`.

Happy coding! 😊